package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "recidiviz.com",
+  "version": "1.0.0",
+  "description": "Real time tracking of recidivism metrics",
+  "private": true,
+  "repository": "git@github.com:Recidiviz/recidiviz.com.git",
+  "author": "Matthew Root <root.matthew@gmail.com>",
```

Comment:
Since it got collapsed elsewhere but is still relevant: is this going to grow into an array of names with further contribution, or should we generalize this to `Team Recidiviz`?

Reply:
Nah, let's change to team. When I was initially doing this I hadn't determined that `team@` was the primary email. Will change to that.